### PR TITLE
Run models in backend

### LINKED ISSUE #18: Run the models in backend and return the actual results

### PR DESCRIPTION
**Pull request by MostafaTwfiq**
Closes #18

### CODE REVIEW

File: src/Backend/ocr_api/models/combined.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        else:
+            ground_truths.append(match.group(1))
+            img = cv2.cvtColor(cv2.imread(str(Path(test_dir) / "images/" / filename)), cv2.COLOR_BGR2RGB)
+            predicted_texts.append(infer_letters(img))
+
+    cer_metric = load_metric("cer")
+    return cer_metric.compute(predictions=predicted_texts, references=ground_truths)
+
+if __name__ == '__main__':
+    print(evaluate_yolo_model("/home/ramez/PycharmProjects/Datagenerator/output/train", word_model))
```
Comment (Youssef1313) [inline src/Backend/ocr_api/models/combined.py:155]: ?

File: src/Backend/ocr_api/models/words.txt
Comment (Youssef1313) [inline src/Backend/ocr_api/models/words.txt:1]: Is this file used?

File: src/Backend/ocr_api/models/punctuation.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import re
```
Comment (Youssef1313) [inline src/Backend/ocr_api/models/punctuation.py:1]: Is this file used?

File: src/Backend/ocr_api/models/combined.py (hunk 1 above)
Comment (MostafaTwfiq) [reply src/Backend/ocr_api/models/combined.py]: 👍

File: src/Backend/ocr_api/models/words.txt
Comment (MostafaTwfiq) [reply src/Backend/ocr_api/models/words.txt:1]: 👍

File: src/Backend/ocr_api/models/punctuation.py (hunk 2 above)
Comment (MostafaTwfiq) [reply src/Backend/ocr_api/models/punctuation.py:1]: We may need it!!

Comment (Youssef1313) [reply src/Backend/ocr_api/models/punctuation.py:1]: We can add it when we do, I think.

Comment (MostafaTwfiq) [reply src/Backend/ocr_api/models/punctuation.py:1]: Ok no problem.

Comment (MostafaTwfiq) [reply src/Backend/ocr_api/models/punctuation.py]: Deleted.

**Pull request merged**